remove svelte ux from tailwind

hahn-kev opened the pull request:
this resolves some style issues around themes

we should probably add backgrounds to the places they were removed (or review them) before merging

myieye reviewed (approved):
Made quite a few changes. The messiest one is that this sort of uncovered another theme issue. I don't understand it entirely, but it had something to do with use riding on the back of svelte-ux's dark mode and something to do with Blazor breaking our mode-watcher after the first load.

![image](https://github.com/user-attachments/assets/4570d607-d018-44da-906e-a551da71dbf6)
![image](https://github.com/user-attachments/assets/125e635f-52a5-45a1-9078-071c9a711040)

![image](https://github.com/user-attachments/assets/861aa2af-70b0-4406-bff4-70c23f641ee7)

![image](https://github.com/user-attachments/assets/7ee8d746-89a4-4607-a4e3-5ff23c43691c)

![image](https://github.com/user-attachments/assets/5d511544-e393-4085-bd53-1e687537df78)

myieye commented:
Actually, there's still a bug on the home page. 😢 
If I'm in dark mode and reload the home page, it fails to load in dark mode.


I'm not sure why it's only the home page. Maybe because the project view takes longer to load 🤷.

hahn-kev commented:
I figured out the home page theme issue. `data-theme` in the html element was getting set to `dark`, it took a while but I was able to track down that it was caused by the `<Settings>` component being used in Server.svelte, after I removed it the issues went way.

I fixed various issues, mostly related to hover effects.

 One thing I think we need to do (out of scope for this PR) is to reduce the amount of stuff in the app bar as there's no room on mobile:
![image](https://github.com/user-attachments/assets/43496246-3350-4e5c-81ce-e4579f6427f2)
obviously `Sandbox` wont be there, but there's not room for the app icon, title, feedback, language and now theme picker. Maybe we just shove that all into a menu?
